data/json/npcs/Backgrounds/gung_ho_1.json

```diff
@@ -22,7 +22,7 @@
   {
     "id": "BGSS_GUNG_HO_1_STORY3",
     "type": "talk_topic",
-    "dynamic_line": "This is it.  This is what I was made for.  There in the street, smashin' monster heads and prayin' I'd make it out?  I've never felt like that.  Alive.  Important.  So after I got myself all stuck back together, I nutted up and went back to it.  Probably killed a thousand Z since then, and I'm still not tired of it.",
```

Comment:
I am not sure about this one either since this is spoken. If this was intended, there might have been an apostrophe, like `Prob'ly` or something.